app/src/main/java/ch/epfl/sdp/blindly/main_screen/MatchPageFragment.kt

```diff
@@ -50,6 +51,9 @@ class MatchPageFragment : Fragment(), CardStackListener {
     @Inject
     lateinit var userRepository: UserRepository
 
+    @Inject
+    lateinit var storage: FirebaseStorage
+
     companion object {
         private const val ARG_COUNT = "matchArgs"
         private var counter: Int? = null
```

Comment:
I know this is from a previous PR but I wonder why a counter was initialized as null.

Reply:
You're right, it's a bit unusual xD I changed it